app/models/wiki_content/revision.rb

```diff
@@ -69,12 +69,18 @@ def plagiarism_report_link
   end
 
   def ref_tags
-    features['feature.wikitext.revision.ref_tags']
+    return features['feature.wikitext.revision.ref_tags'] if
```

Comment:
This could instead be something like `features['feature.wikitext.revision.ref_tags'] || features['feature.len(<datasource.wikidatawiki.revision.references>)']`.

Also, the key for each feature could probably be turned into a constant to make it clearer:

`features[WIKITEXT_REF_TAGS] || features[WIKIDATA_REFERENCES]`